### ISSUE TITLE
As17290 Peer Request

### ISSUE THREAD

**Issue by ahadley1124** [routeviews/public-issues#172]
### Full Name

Austin Hadley, sole proprietor

### ASN

~~17299~~
17290

### PeeringDB Integration

- [X] My PeeringDB Entries are current
- [ ] Peer at **ALL EXCHANGES** (*and skip the Peer Address(es) field below*)

### Peer Address(es)

172.82.26.82
2605:9FC0:179:6::2

### Multihop

- [X] Multihop Peering Request

**Comment by pfsinoz** [comment]
I don't see any PeeringDB entry for AS17299.  https://www.peeringdb.com/asn/17299 gives page not found. Nor do I see AS17299 in the BGP table (hasn't been announced in >8 years). Please provide more information about your request. Or is it AS17290 as per in the title?

**Comment by ahadley1124** [comment]
My apologies, should be AS17290

**Comment by nihb333** [comment]
Hi - We see one  v4 prefix announced to one upstream and very limited number of peers.  So while we are happy for your support, we would like to know about more about your network before agreeing to peering. Will you be able to announce your full table to us on the session?

**Comment by ahadley1124** [comment]
Yes I will be able to announce the full route table. I currently have 2 V6 prefixes that I'm working to get announced. The local IX is in the process of onboarding 5 new members which will significantly increase route table view. Please let me know what other information you would like.

**Comment by nihb333** [comment]
What kind of network are you running?  I see you listed as a dsl/cable provider in peeringdb, but your website links to a virtuel hosting login, so this is part of our confusing about what kind of network you operate and what the value of a peering would be 
Thanks

**Comment by nihb333** [comment]
requester went silent
